src/components/PressableWithSecondaryInteraction/index.js

```diff
@@ -58,6 +62,9 @@ class PressableWithSecondaryInteraction extends Component {
                 style={StyleUtils.combineStyles(this.props.inline ? styles.dInline : this.props.style)}
                 onPressIn={this.props.onPressIn}
                 onLongPress={(e) => {
+                    if (!this.props.onSecondaryInteraction) {
```

Comment:
I don't think this is going to work (it didn't during my testing), we need to set onLongPress to undefined or null, otherwise it's still vibrates

Reply:
No, on native we're calling .vibrate manually, so it's not needed

Reply:
Already pushed it up. I can roll it back with another commit if necessary, but it still achieves the same result.

Reply:
Let's roll it back, having an early return is a bit more readable